include/translatable-object-with-types-trait.php

```diff
@@ -34,15 +34,19 @@ trait PLL_Translatable_Object_With_Types_Trait {
 	 *
 	 * @param int[] $language_ids List of language `term_taxonomy_id`.
 	 * @param int   $limit        Max number of objects to return. `-1` to return all of them.
+	 * @param array $args {
+	 * 		The object args.
+	 *
+	 * 		@type string|string[] $object_types A translated object type or an array of translated object types.
+	 * }
 	 * @return string
 	 *
 	 * @phpstan-param array<positive-int> $language_ids
 	 * @phpstan-param -1|positive-int $limit
+	 * @phpstan-param array<array<string, string>> $args
```

Comment:
We can be more precise, to force the `type` key:
```suggestion
	 * @phpstan-param array{
	 *     type?: string|array<string>
	 * } $args
```
I'd like to use `non-empty-string` here instead of `string` but it would require to improve the documentation of `PLL_Model::get_posts_with_no_lang()`, `PLL_Model::get_terms_with_no_lang()`, and `PLL_Model::get_translated_post_types()`. So, maybe another time.

Reply:
We still need this one. I didn't meant to say we should remove the tag, just to forget `non-empty-string` for now.